### DISCUSSION TITLE
Is it possible to change the delimiter in --plain ?

### DISCUSSION THREAD

**Discussion by iksi4prs** [ankitpokhrel/jira-cli#658] [General]
Hi,
I want to use the tool also to save data to csv,
but when using the default, the output can contain more than on TAB between columns (depends on width of columns),
so make it hard to do plain parse/import.

Is it possible now ? 

Thanks,
Si.

**Comment by arkadiusnowakowski** [comment]
Maybe a new  parameter "--csv" could be implemented.
This would also help if, custom fields could be exported using "--csv".

**Reply by iksi4prs** [reply]
@arkadiusnowakowski , I've created a PR where you can use any delimiter.
You can find it here:
https://github.com/ankitpokhrel/jira-cli/pull/662

It didn't pass the automatic tests because I didn't download the testing system to check it against,
but the feature works well for me.

Also I'm not sure  what is the status of this project because there are few old PRs that are still open.
You can get my branch from my PR and use it.
